www/src/config.ts

```diff
@@ -139,6 +165,11 @@ export const SIDEBAR_HEADER_MAP: Record<
   Exclude<KnownLanguageCode, "en">,
   Record<OuterHeaders, string>
 > = {
+  es: {
+    "Create T3 App": "Create T3 App",
```

Comment:
```suggestion
    "Create T3 App": "Crear T3 App",
```

Reply:
I wouldn't change this. "Create T3 App" is a name so it shouldn't be translated.

Reply:
Agreed

Reply:
👍

Reply:
This one is still pending ?